src/platform-includes/source-context/java.mdx

```diff
@@ -310,3 +352,21 @@ sentry.bundle-ids=A_VALID_UUID`
 ```Java
 options.addBundleId("A_VALID_UUID");
 ```
+
+<PlatformSection supported={["android"]}>
+
+## Manually Uploading Native Symbols and Source Code
```

Comment:
```suggestion
## Manually Uploading Debug Symbols and Source Code
```

Reply:
I'd also change every place you mention `native symbols` to `debug symbols` in this page.